### PR TITLE
Xw 542

### PR DESCRIPTION
**Pull request by justnpT**
https://wikia-inc.atlassian.net/browse/XW-542

changed goToCurrentArticleEditPage to editArticleWithQueryParam
replaced editWithDropdown with editArticleWithQueryParam wherever possible
resolving action=edit so that it is reused in the repository

### CODE REVIEW

**Comment by vforge** [comment]
LGTM

File: src/test/java/com/wikia/webdriver/testcases/admindashboardtests/EditingLocalCssTests.java [hunk 1]
```diff
@@ -43,12 +43,12 @@ public void EditingLocalCss_002_UserWithoutAdminRightsHasNoEditOption() {
     WikiBasePageObject base = new WikiBasePageObject(driver);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/admindashboardtests/EditingLocalCssTests.java]: MAJOR Remove this unused "base" local variable. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 31 issues:
* MAJOR 17 major
* MINOR 14 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L747) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L738) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Move this trailing comment on the previous empty line.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L564) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)
* MINOR [Define a constant instead of duplicating this literal "page loads for more than 30 seconds" 3 times.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L739) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L571) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L952) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "logOut" 5 times.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L739) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Move this trailing comment on the previous empty line.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L569) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)
* MINOR [Define a constant instead of duplicating this literal " opened" 5 times.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L214) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "%userName%" 4 times.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L321) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Remove those useless parentheses.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L696) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)
* MINOR [Move the array designator from the variable to the type.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L447) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1197)
* MINOR [Reorder the modifiers to comply with the Java Language Specification.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L94) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L757) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L966) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "DropdownVerified" 3 times.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java#L396) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Add a default case to this switch.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java#L801) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
* MINOR [Replace this "switch" statement by "if" statements to increase readability.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java#L899) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1301)
* MAJOR [Add a default case to this switch.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java#L861) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
* MAJOR [Add a default case to this switch.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java#L841) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
* MAJOR [Add a default case to this switch.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java#L821) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
* MAJOR [Make this final field static too.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java#L169) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialCssPageObject.java#L221) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "the following text was send to ace editor: " 3 times.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialCssPageObject.java#L112) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleFeaturesCRUDTestsAnon.java#L11) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00115)
* MINOR [Define a constant instead of duplicating this literal "image" 10 times.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleFeaturesCRUDTestsUser.java#L49) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [4 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleFeaturesCRUDTestsUser.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MINOR [Move this trailing comment on the previous empty line.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/testcases/mediatests/lightboxtests/LightboxTests.java#L87) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)
* MAJOR [This block of commented-out lines of code should be removed.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/testcases/mediatests/lightboxtests/LightboxTests.java#L159) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
* MINOR [Reorder the modifiers to comply with the Java Language Specification.](https://github.com/Wikia/selenium-tests/blob/447377d06464e0d267e82ef1f2301e2030e9c34f/src/test/java/com/wikia/webdriver/testcases/mediatests/modal/VetModalWidthTests.java#L30) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)

**Comment by bognix** [comment]
LGTM

**Pull request merged**